CMakeLists.txt

```diff
@@ -1,10 +1,15 @@
 # CMakeLists.txt has to be located in the project folder and cmake has to be
 # executed from 'project/build' with 'cmake ../'.
-cmake_minimum_required(VERSION 2.6)
+cmake_minimum_required(VERSION 3.1)
+project(vizkit3d_debug_drawings VERSION 0.1)
+
 find_package(Rock)
-set(CMAKE_CXX_STANDARD_REQUIRED 11)
 set(ROCK_TEST_ENABLED ON)
-rock_init(vizkit3d_debug_drawings 0.1)
+
+rock_init()
 rock_find_qt4()
```

Comment:
I guess in contrast to the qt4-qt5 branch this does not intent to support Ubuntu > 20.04 yet?

Reply:
That is correct, once this has been ironed out, i will move the qt4/qt5 branch accordingly.